vendor/github.com/openshift/source-to-image/pkg/build/strategies/sti/sti.go

```diff
@@ -562,7 +562,6 @@ func (builder *STI) Execute(command string, user string, config *api.Config) err
 		}()
 
 		opts.Stdin = r
-		defer wg.Wait()
```

Comment:
@rhcarvalho FYI ^^ from @liggitt re: the re-write you mentioned in https://github.com/openshift/source-to-image/pull/581